src/bundler/linker.ts

```diff
@@ -26,13 +26,28 @@ export function link(
     assets: assetMaps.assets ?? {},
     webviewAssets: assetMaps.webviewAssets ?? {},
     code: es,
+    sourceMap: extractInlineSourceMap(es),
     hostname,
     products: {},
     provides: provides(),
     uses: uses()
   }
 }
 
+/**
+ * Extracts the inline source map generated by tsc.
+ * @arg es code blob that's expected to include an inline source map
+ * @returns the source map as a JSON-encoded string
+ */
+function extractInlineSourceMap(es: string): string {
+  const lastLine = es.split('\n').pop()
```

Comment:
:no_bell: at(-1)
:no_bell: might be worth checking a file with windows line endings too.